<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>600</width>
    <height>0</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">#MainWindow {
    background-color: #2f2f2f; /* Основной фон окна */
}

QMenuBar {
    background-color: #4f4f4f; /* Фон меню-бара */
    color: #ffffff; /* Цвет текста в меню-баре */
}

QMenuBar::item {
    background-color: transparent; /* Фон пунктов меню */
    color: #ffffff; /* Цвет текста пунктов меню */
}

QMenuBar::item:selected {
    background-color: #6f6f6f; /* Фон выбранного пункта меню */
}

QMenu {
    background-color: #4f4f4f; /* Фон выпадающих меню */
    color: #ffffff; /* Цвет текста в выпадающих меню */
}

QMenu::item {
    background-color: transparent; /* Фон пунктов выпадающих меню */
    color: #ffffff; /* Цвет текста пунктов выпадающих меню */
}

QMenu::item:selected {
    background-color: #6f6f6f; /* Фон выбранного пункта выпадающего меню */
}

QPushButton {
    background-color: #4f4f4f; /* Фон кнопок */
    color: #ffffff; /* Цвет текста на кнопках */
    border: 1px solid #6f6f6f; /* Граница кнопок */
    padding: 5px; /* Отступы текста на кнопках */
}

QPushButton:hover {
    background-color: #6f6f6f; /* Фон кнопок при наведении */
}

QPushButton:pressed {
    background-color: #3f3f3f; /* Фон кнопок при нажатии */
}

QLabel {
    color: #ffffff; /* Цвет текста в метках */
}

QLineEdit {
    background-color: #4f4f4f; /* Фон полей ввода */
    color: #ffffff; /* Цвет текста в полях ввода */
    border: 1px solid #6f6f6f; /* Граница полей ввода */
    padding: 5px; /* Отступы текста в полях ввода */
}

QComboBox {
    background-color: #4f4f4f; /* Фон комбо-боксов */
    color: #ffffff; /* Цвет текста в комбо-боксах */
    border: 1px solid #6f6f6f; /* Граница комбо-боксов */
    padding: 5px; /* Отступы текста в комбо-боксах */
}

QComboBox QAbstractItemView {
    background-color: #4f4f4f; /* Фон выпадающего списка комбо-бокса */
    color: #ffffff; /* Цвет текста в выпадающем списке комбо-бокса */
}

QComboBox QAbstractItemView::item {
    background-color: transparent; /* Фон пунктов выпадающего списка комбо-бокса */
    color: #ffffff; /* Цвет текста пунктов выпадающего списка комбо-бокса */
}

QComboBox QAbstractItemView::item:selected {
    background-color: #6f6f6f; /* Фон выбранного пункта выпадающего списка комбо-бокса */
}

QGroupBox::title {
    color: white;
}

QMessageBox{
	background-color: #2f2f2f;
}</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout_4" columnstretch="20,80">
    <item row="0" column="0">
     <widget class="QGroupBox" name="groupBox">
      <property name="title">
       <string>Генерация пула игр</string>
      </property>
      <property name="flat">
       <bool>false</bool>
      </property>
      <layout class="QGridLayout" name="gridLayout">
       <item row="5" column="1">
        <widget class="QSpinBox" name="gameCountBox">
         <property name="minimum">
          <number>100</number>
         </property>
         <property name="maximum">
          <number>99999</number>
         </property>
        </widget>
       </item>
       <item row="5" column="0">
        <widget class="QSpinBox" name="playerCountBox">
         <property name="minimum">
          <number>10</number>
         </property>
         <property name="maximum">
          <number>9999</number>
         </property>
        </widget>
       </item>
       <item row="4" column="1">
        <widget class="QLabel" name="label_4">
         <property name="text">
          <string>Количество игр</string>
         </property>
        </widget>
       </item>
       <item row="1" column="0" colspan="2">
        <widget class="QDateTimeEdit" name="startDateBox"/>
       </item>
       <item row="0" column="0">
        <widget class="QLabel" name="label">
         <property name="text">
          <string>Игры начнутся</string>
         </property>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QLabel" name="label_2">
         <property name="text">
          <string>Игры закончатся</string>
         </property>
        </widget>
       </item>
       <item row="4" column="0">
        <widget class="QLabel" name="label_3">
         <property name="text">
          <string>Количество игроков</string>
         </property>
        </widget>
       </item>
       <item row="3" column="0" colspan="2">
        <widget class="QDateTimeEdit" name="endDateBox">
         <property name="dateTime">
          <datetime>
           <hour>0</hour>
           <minute>0</minute>
           <second>0</second>
           <year>2001</year>
           <month>1</month>
           <day>21</day>
          </datetime>
         </property>
         <property name="minimumDateTime">
          <datetime>
           <hour>0</hour>
           <minute>0</minute>
           <second>0</second>
           <year>2001</year>
           <month>1</month>
           <day>21</day>
          </datetime>
         </property>
        </widget>
       </item>
       <item row="9" column="1">
        <widget class="QPushButton" name="pushButton_2">
         <property name="text">
          <string>Начать симуляцию</string>
         </property>
        </widget>
       </item>
       <item row="8" column="0" colspan="2">
        <widget class="QSpinBox" name="playersInTeamBox">
         <property name="minimum">
          <number>2</number>
         </property>
        </widget>
       </item>
       <item row="6" column="0" colspan="2">
        <widget class="QLabel" name="label_5">
         <property name="text">
          <string>Количество игроков в 1 команде</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item row="0" column="1" rowspan="3">
     <layout class="QGridLayout" name="gridLayout_2">
      <item row="0" column="0">
       <widget class="QTableView" name="tableView">
        <property name="styleSheet">
         <string notr="true">QTableView {
        background-color: #2f2f2f;
        color: white;
        selection-background-color: #333;
        selection-color: white;
    }
    
    QTableView::item {
        padding: 5px;
    }
    
    QTableView::item:selected {
        background-color: #333;
        color: white;
    }
    
    QHeaderView::section {
        background-color: #2f2f2f;
        color: white;
        padding: 5px;
    }
    
    QHeaderView::section:hover {
        background-color: #333;
        color: white;
    }</string>
        </property>
        <property name="selectionBehavior">
         <enum>QAbstractItemView::SelectItems</enum>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <layout class="QHBoxLayout" name="horizontalLayout" stretch="80,20">
        <item>
         <widget class="QComboBox" name="comboBox"/>
        </item>
        <item>
         <widget class="QPushButton" name="pushButton">
          <property name="text">
           <string>Импорт БД</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
    <item row="1" column="0">
     <layout class="QGridLayout" name="gridLayout_3">
      <item row="0" column="0">
       <widget class="QPushButton" name="analyzeButton">
        <property name="text">
         <string>Проанализировать точность распределения</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="meanRatingLabel">
        <property name="text">
         <string>Средний рейтинг:</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="stdDevLabel">
        <property name="text">
         <string>Cтандартное отклонение:</string>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QLabel" name="medianRatingLabel">
        <property name="text">
         <string>Медиана:</string>
        </property>
       </widget>
      </item>
      <item row="4" column="0">
       <widget class="QTextEdit" name="analysisTextEdit">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>100</height>
         </size>
        </property>
        <property name="readOnly">
         <bool>true</bool>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="2" column="0">
     <spacer name="verticalSpacer">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>20</width>
        <height>251</height>
       </size>
      </property>
     </spacer>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
